csds-client/client/util.go

```diff
@@ -194,7 +199,7 @@ func parseConfigStatus(xdsConfig []*csdspb.PerXdsConfig) []string {
 }
 
 // printOutResponse processes response and print
-func printOutResponse(response *csdspb.ClientStatusResponse, fileName string) error {
+func printOutResponse(response *csdspb.ClientStatusResponse, fileName string, visualization bool, monitor bool) error {
```

Comment:
Add some point we should have a struct holding these booleans.

Reply:
Modified this to `c.info`